src/main/java/io/javalin/apibuilder/ApiBuilder.java

```diff
@@ -488,14 +488,16 @@ public static void crud(@NotNull String path, @NotNull CrudHandler crudHandler,
         String SEPARATOR = "/:";
         String resourceBase = path.substring(0, path.lastIndexOf(SEPARATOR));
         String resourceId = path.substring(path.lastIndexOf(SEPARATOR) + SEPARATOR.length());
+        final Map<CrudHandlerType, Handler> handlerMap = crudHandler.asMap(resourceId);
 
-        Map<CrudHandlerLambdaKey, Handler> lambdas = CrudHandlerKt.getLambdas(crudHandler, resourceId);
-        lambdas = OpenApiBuilder.documented(crudHandler, lambdas);
+        staticInstance().get(prefixPath(path), handlerMap.get(CrudHandlerType.GET_ONE), permittedRoles);
```

Comment:
Looks similar, but the ApiBuilder does not care if the crud interface is documented or not.